pulumi/infra/backup.py

```diff
@@ -0,0 +1,61 @@
+import json
```

Comment:
All this files with the features should be refactored to the pulumi components as it was in the example that I've provided